sdk/src/main/java/com/ifountain/opsgenie/client/AbstractOpsGenieHttpClient.java

```diff
@@ -126,8 +127,63 @@ protected BaseResponse doGetRequest(BaseRequest request) throws OpsGenieClientEx
         return populateResponse(request, httpResponse);
     }
 
+    protected BaseResponse doPostRequestV2(BaseRequest request) throws IOException, OpsGenieClientException, ParseException {
```

Comment:
Could we not remove the old version of the methods completely and skip the version prefix. The old methods aren't functional anyway right?

Reply:
We are releasing clients one by one. So We can rename these methods along with the last client to be released.

Reply:
Got it. So the missing file for instance, would be added in another PR? I guess I should be reviewing them in a particular order then? I picked this one up as it was unreviewed.

Reply:
I have added all the missing files in this PR too.